### PR TITLE
fix: getTtlMs failing with cookie.expires only

### PR DESCRIPTION
**Pull request by RenanGalvao**
The first time ``cookie.expires`` passes through the ``getTtlMs`` function it is a ``Date`` object, so the ``Number(expires)`` conversion works and returns the same value as ``new Date(expires).getTime()``, the problem occurs when the ``expires`` value is retrieved from the store, it is no longer a Date object, in reality it is a ``string``, which causes an error when the conversion to number is done.

I've included some tests. Nice module.

### CODE REVIEW

File: src/lib/session.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ const getTtlMs = (cookie: CookieSerializeOptions): number => {
 
 	if (maxAge) return maxAge * 1000;
 	if (expires) {
-		const ms = Number(expires) - Date.now();
+		const ms = new Date(expires).getTime() - Date.now();
```
Comment (yutak23) [inline src/lib/session.ts:39]: Thank you, I was omitting to take into account that the `Date `object serialized with `JSON.stringify()` is a string.

**Review by yutak23 — APPROVED** [review]

**Review by yutak23 — CHANGES_REQUESTED** [review]
There seems to be a formatting problem. I apologize for the inconvenience, but please correct the problem.
image

File: tests/integration/expires.test.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import type { SessionData } from '$lib/index.js';
```
Comment (yutak23) [inline tests/integration/expires.test.ts:1]: There seems to be an eslint error
Would you correct it?

image

**Review by yutak23 — CHANGES_REQUESTED** [review]

File: tests/integration/expires.test.ts (hunk 2 above)
Comment (RenanGalvao) [reply tests/integration/expires.test.ts]: Yep. Sorry for those little problems.

**Review by yutak23 — APPROVED** [review]
thank you!

**Comment by yutak23** [comment]
thanks a lot @RenanGalvao, this has been released as v0.3.2

**Pull request merged**